docs/src/pages/components/searchfield/react.mdx

```diff
@@ -15,20 +21,23 @@ clear button. When users hit `Enter` key or click the search icon, the `onSubmit
 
 ## Usage
 
-Import the `SearchField` component and styles and provide a `label` for accessibility/usability.
+Import the `SearchField` primitive, and provide a `label` for accessibility/usability.
 
-```jsx
-import { SearchField } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <View>
```

Comment:
I was wondering if we really need to add a `View` wrapper here. It seems unnecessary